### ISSUE TITLE
Impossible to plot Insample Anomalies without a `forecast_df`

### ISSUE THREAD

**Issue by omriKramer** [Nixtla/utilsforecast#97]
Right now it is Impossible to plot anomalies for insample data without passing a `forecast_df` even though it is a valid use case. Moreover, the prediction intervals for `df` are filtered out by `df = _filter_series(...)` at line 162 which prevents plotting the insample anomalies.

**Comment by jmoralez** [comment]
The following should work:

```python
from utilsforecast.data import generate_series
from utilsforecast.plotting import plot_series

level = [95]
series = generate_series(2, n_models=1, level=level)
plot_series(series, series, level=level, plot_anomalies=True, max_insample_length=0)
```

**Comment by omriKramer** [comment]
You are right, didn't realize I can use `max_insample_length` thank you!

**Comment by jmoralez** [comment]
We can make the df optional and being able to provide just forecasts_df for this case to avoid having to repeat it

**Comment by jmoralez** [comment]
Fixed by #98
